src/Test.UnitTests.BinaryParsers/PEBinary/PEBinaryTests.cs

```diff
@@ -102,18 +102,47 @@ public void PEBinary_PdbIsStripped()
         }
 
         [Fact]
-        public void PEBinary_ContainsExpectedLanguageCode()
+        public void PEBinary_NativeBinaryContainsExpectedLanguageCode()
         {
             if (!PlatformSpecificHelpers.RunningOnWindows()) { return; }
 
             ContainsLanguageCode("clangcl.pe.c.codeview.exe", Language.C).Should().BeTrue();
             ContainsLanguageCode("clangcl.pe.cpp.codeview.exe", Language.Cxx).Should().BeTrue();
             // As of v1.58.1 Rust official compiler RustC does not yet use the new CV_CFL_LANG code for Rust.
-            // We will have another test file when new version use it.
             ContainsLanguageCode("Native_x64_RustC_Rust_debuginfo2_v1.58.1.exe", Language.Rust).Should().BeFalse();
+            // As of v1.67.1 Rust official compiler RustC already use the new CV_CFL_LANG code for Rust.
+            ContainsLanguageCode("Native_x64_RustC_Rust_debuginfo2_v1.67.1.exe", Language.Rust).Should().BeTrue();
             ContainsLanguageCode("Native_x64_VS2019_CPlusPlus_DEBUG_DEFAULT.dll", Language.Cxx).Should().BeTrue();
         }
 
+        [Fact]
+        public void PEBinary_IsManaged()
+        {
+            if (!PlatformSpecificHelpers.RunningOnWindows()) { return; }
+
+            IsManaged("clangcl.pe.c.codeview.exe").Should().BeFalse();
+            IsManaged("clangcl.pe.cpp.codeview.exe").Should().BeFalse();
+            IsManaged("Native_x64_RustC_Rust_debuginfo2_v1.67.1.exe").Should().BeFalse();
+            IsManaged("Native_x64_VS2019_CPlusPlus_DEBUG_DEFAULT.dll").Should().BeFalse();
+            IsManaged("Managed_x64_VS2022_CSharp_Net48_Default.exe").Should().BeTrue();
+            IsManaged("Managed_x64_VS2022_CSharp_Net70_Default.exe").Should().BeFalse();
+            IsManaged("Managed_x64_VS2022_CSharp_NetCore31_Default.exe").Should().BeFalse();
+        }
+
+        [Fact]
+        public void PEBinary_IsDotNetCore()
+        {
+            if (!PlatformSpecificHelpers.RunningOnWindows()) { return; }
+
+            IsDotNetCore("clangcl.pe.c.codeview.exe").Should().BeFalse();
+            IsDotNetCore("clangcl.pe.cpp.codeview.exe").Should().BeFalse();
+            IsDotNetCore("Native_x64_RustC_Rust_debuginfo2_v1.67.1.exe").Should().BeFalse();
+            IsDotNetCore("Native_x64_VS2019_CPlusPlus_DEBUG_DEFAULT.dll").Should().BeFalse();
+            IsDotNetCore("Managed_x64_VS2022_CSharp_Net48_Default.exe").Should().BeFalse();
+            IsDotNetCore("Managed_x64_VS2022_CSharp_Net70_Default.exe").Should().BeFalse();
+            IsDotNetCore("Managed_x64_VS2022_CSharp_NetCore31_Default.exe").Should().BeFalse();
```

Comment:
All these tests passed in this PR.
strange that even the NetCore31 returned false, should have been tested.
Wonder if I missed something.